zstacklib/zstacklib/gpu/vendors/huawei.py

```
@@ -0,0 +1,438 @@
+            # Store NPU ID mapping
+            for info in gpu_infos:
+                if info.pci_address:
+                    npu_id_map[info.pci_address.lower()] = npu_id
+                    # Store NPU ID in extra field
+                    info.extra["npuId"] = npu_id
+                    
+                    # Check isolation status
+                    try:
+                        is_isolated = cls.check_npu_isolation(npu_id, npu_ids)
+                        info.extra["isIsolated"] = is_isolated
+                    except Exception as ex:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)